app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
   </head>
   <body>
     <div class="container-fluid">
+      <%= render 'layouts/header' %>
```

Comment:
If you rebase your PR, a new file will show up, `spec/views/layouts/application_spec.rb`
We should probably include this render call in the spec